pkg/probe/firewall_load_balance.go

```diff
@@ -133,11 +133,13 @@ func probeFirewallLoadBalance(c http.FortiHTTP, meta *TargetMetadata) ([]prometh
 				}
 
 				realServerRTTValue := math.NaN()
-				if "<1" == realServer.RTT {
+				switch realServer.RTT {
+				case "<1":
 					realServerRTTValue = 0.001
-				} else if "" == realServer.RTT {
+				case "":
 					// NaN
-				} else {
+					fallthrough
```

Comment:
`fallthrough` changes this behavior doesn't it? Before the default case code would only execute when RTT was NOT "<1" or ""

Reply:
I think you're correct, not sure what I was thinking with this.